static/css/devhub/listing.less

```diff
@@ -166,3 +169,7 @@
 #addon-delete-listing.item:hover a.delete-addon {
     color: #fff;
 }
+
+.item:hover button.link {
+    color: #0996F8;
+}
```

Comment:
you need to have an empty line at the bottom of all the files.

Reply:
@eviljeff  amended the commit to add a line to it